src/openrct2/drawing/NewDrawing.cpp

```diff
@@ -233,18 +233,18 @@ void gfx_draw_dashed_line(
     }
 }
 
-void FASTCALL gfx_draw_sprite(rct_drawpixelinfo* dpi, ImageId image_id, const ScreenCoordsXY& coords)
+void FASTCALL gfx_draw_sprite(rct_drawpixelinfo* dpi, int32_t image, const ScreenCoordsXY& coords, uint32_t tertiary_colour)
```

Comment:
I guess we still need this overload?

Reply:
There are hundreds of calls to this, will take a long time to go through them all. Most important ones are just any that draw an image from an object file. UI drawing isn't so important as those IDs will never be above the old image limit.

Reply:
Actually it seems many have already been changed, there are still a few left though. Can be removed once they are all changed.